src/applications/personalization/rated-disabilities/containers/RatedDisabilitiesApp.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import backendServices from 'platform/user/profile/constants/backendServices';
+import { fetchRatedDisabilities } from '../actions';
+
+// Wonder if we can put RD data in platform...
+import RequiredLoginView from 'platform/user/authorization/components/RequiredLoginView';
+import RatedDisabilityView from '../components/RatedDisabilityView';
+// import RatedDisabilityHeader from '../components/RatedDisabilityHeader';
+
+class RatedDisabilitiesApp extends React.Component {
+  render() {
+    const { ratedDisabilities } = this.props.ratedDisabilities;
```

Comment:
Same issue with a potential runtime error here. Check out [this potential solution]() 👍